index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Theophilus Doku</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <!-- Header -->
+    <header class="header">
+            <a href="#" class="logo">
+                Sozo
+            </a>
+            <button class="menu_btn" type="button">
+                <img src="images/menu.png" alt="menu-button">
+            </button>
```

Comment:
- Kindly use for the Hamberger button a `nav` tag for semantics reasons as your previous code reviewer suggested and make the logo `a outside the `nav`.👌
```
<header class="header">
            <a href="#" class="logo">
                Sozo
            </a>
           <nav>
              <button class="menu_btn" type="button">
                  <img src="images/menu.png" alt="menu-button">
              </button>
            </nav>
</header>
```
👌👍
- Kindly minimize the Humber icon size to fit the Figma design as the previous code reviewer suggested you 👌👍